pkg/settings/settings.go

```diff
@@ -0,0 +1,42 @@
+package settings
+
+import (
+	"errors"
+	"os"
+	"strconv"
+)
+
+// Global
+var Settings = _Settings{}
```

Comment:
Is there meaning behind the `_Settings` type with a leading underscore? Haven't seen that notation in golang before.

Reply:
IIRC, I saw that in kubernetes code.  It's a way to avoid the name collision between the global var and the type without using a wonky name for either.  I could have made the type lowercase but really don't like making types lowercase.

Reply:
Huh, good idea. Thanks.